kratos/geometries/triangle_2d_6.h

```diff
@@ -660,22 +660,21 @@ template<class TPointType> class Triangle2D6
         // faces in columns
         if(NodesInFaces.size1() != 4 || NodesInFaces.size2() != 3)
             NodesInFaces.resize(4,3,false);
-
-	//compatible to Edges() function. Triangle library considers a different order
-        NodesInFaces(0,0)=0;//face or master node
+        //face 1
+        NodesInFaces(0,0)=0;//contrary node to the face
         NodesInFaces(1,0)=1;
-        NodesInFaces(2,0)=4;
-        NodesInFaces(3,0)=2;
- 
-        NodesInFaces(0,1)=1;//face or master node
+        NodesInFaces(2,0)=2;
+        NodesInFaces(3,0)=4;//quadratic nodes after edge nodes
```

Comment:
I think you are inverting these (same below)

Reply:
The commentary says itself "quadratic nodes after edge nodes"

Reply:
Yes there is an error here, let me correct it

Reply:
In any case, @josep-m-carbonell can you create a test for the geometries in cpp as t is in the folder tests?

Reply:
Yes, that's right. It must be like this. First the  edges and then the other nodes, it is just a convention, I think only @AFranci was using it, he can say something about

Reply:
In fact I didn't want to merge this change in the PR. I forgot that this change was made. I will revert it.